### ISSUE TITLE
Merging your changes in Pipedal mainline

### ISSUE THREAD

**Issue by rerdavies** [FoolHen/pipedal#4]
I'm just about to push  a Pipedal 1.2 commit with bookworm support on my repository. It's a huge push (almost 200 files changed, don't ask, the TLDR, Wifi P2P is not supported by the NetworkManager networking stack, so drastic steps were required). The push should happen before end-of-day. 

I have done my best to included your commits, although I have manually merged rather than doing it through a pull request. I'm still not entirely comfortable with github, and it seemed easier to just do it rather than using pull requests workflow to negotiate changes on the commits that I didn't like. I promise to use github better (proper branching would have made it easier to pull the commits) in future work. :-/

I would also like to add your copyright to files affected. It matters, because any copyrights in the source are automatically added to the open-source copyright notices in the about box. And special priority is given to contributors to the project. The format for the notice should be either:

     Copyright (c) Your Name   

(current best practice) or 

   Copyright (c) 2024 Your Name

(because omitting the year grates with those of us who grew up under old copyright regimes). Both forms are legally correct.

I'm assuming your updates are licensed under the same MIT license I'm using. Please say so in your reply. If you really want to use a different license, that can be accomodates, as long as your license isn't GPL.

Let me know what copyright notice you'd like to use.

The following commits have been manually merged, pretty much verbatim:

[Fix GxTuner never displaying notes](https://github.com/rerdavies/pipedal/commit/53ff6a1dc2e224044dacc0fbbf02bcc3af9e6860)

[Fix banks white bg in dark mode](https://github.com/rerdavies/pipedal/commit/813844395f2067eaa5a1c39ac8fdbe45727a83d9)

[Fix socket port default config value](https://github.com/rerdavies/pipedal/commit/d0e44bfe2203c8b3acfa985626490fb87b239e5d)

[Fix bypass with midi not affecting audio](https://github.com/rerdavies/pipedal/commit/360b8b0f97624573cf1d1f4288360f7291edefe6)

---
[Fix uninstall script](https://github.com/rerdavies/pipedal/commit/d8d05d68918323562d6f7da71175f7804714d9b6)

- Your fixes included (plus additional directories cleaned), but the file has been renamed to install.sh to mach the convention used by other scripts. The original wasn't truly functional. But now it is.

[Fix type error in PresetSelector](https://github.com/rerdavies/pipedal/commit/7c5a816904aebde7375033a05c923a615872a30b)

- I have rejected this change. I think the original code is correct. the type of value is number | string | undefined, and the documention specifies that "" means that no value is selected. Let me know if you disagree. My concern is that converting the instanceId to string will mean that subsequent === comparisons will fail in the onclick handler. 

[Fix some errors in docs](https://github.com/rerdavies/pipedal/commit/9c476c4da7dd9acaab51f4f726fddeda57c8735f)

- Partially accepted. Stripping documentation relating to VS Code was rejected.

[Fix ToobML gain control not disabling](https://github.com/rerdavies/pipedal/commit/4a39670aea657e77a4344bfe075b0ed97e7b9177)

- Postponed until I can commit my changes. I need to run your code through VS Code auto-formatter to determine what the actual changes were. And the original code is lethally dangerous. :-/ It's possible that I may do some serious rewriting to make it safer. I'll get working on merging your commit as soon as I can.

**Comment by maarthome** [comment]
Hi Robin,

Nice to see that you are back.... alive and kicking!!!

Regards,
Maarten

**Comment by rerdavies** [comment]
What copyright notice would you like to apply?

**Comment by FoolHen** [comment]
Hi @rerdavies , sorry I never replied, but better late than never. I guess I missed the notification back then. Sure, if you don't mind you can add me as  `Copyright (c) Fulgencio Ruiz Rubio`.

**Comment by rerdavies** [comment]
Added. Will probably show up when PiPedal 2.0 gets merged onto main. And it may take a while to propagate into the About Box credits. But it will. 

Thanks again for the contributions.
